resolwe_bio/tools/basespace_download.py

```diff
@@ -51,8 +72,8 @@ def main():
         headers = {'x-access-token': access_token}
 
         for file_id in file_ids:
-            file_name = get_file_name(session, file_id, headers)
-            download_file(session, file_id, file_name, headers)
+            file_name, file_size = get_file_properties(session, file_id, headers)
+            download_file_repeatedly(args.tries, session, file_id, file_name, file_size, headers)
             output(args.output, 'filename={}'.format(file_name))
     except Exception:
         if args.verbose:
```

Comment:
You shouldn't hide the error.

Reply:
This is a security measure. You don't want to print anything because of the secrets. There's a verbose argument for this reason.

Reply:
Can you give me an example when a secret is printed?

Reply:
You don't need to print the whole traceback, but the error message should definitely be printed.

Reply:
I can't, but it's a possibility. Kostko was pretty adamant about it.

Reply:
It's a broad exception catch so you don't know what going to be printed. There could be sensitive information even in the message.

Reply:
I am pretty sure that Python never prints any values in tracebacks - at most it reveals dict's keys. And I am totally certain that it doesn't print anything sensitive in the error message if you don't put it there 🙂

The user should know why the download failed, so you need to print the actual message.